backend/shared/types.d.ts

```diff
@@ -1,16 +1,16 @@
-import { DynamoDB } from 'aws-sdk';
-import { PostToConnectionRequest } from 'aws-sdk/clients/apigatewaymanagementapi';
+import { PostToConnectionCommandInput } from '@aws-sdk/client-apigatewaymanagementapi/dist-types/commands/PostToConnectionCommand';
+import { DynamoDBDocument } from '@aws-sdk/lib-dynamodb';
 import { CardValue } from '../../shared/cards';
 
 export interface Config {
   connectionId: string;
   tableName: string;
-  ddb: DynamoDB.DocumentClient;
+  ddb: DynamoDBDocument;
 }
 
 export interface ConfigWithHandler extends Config {
   handler: {
-    postToConnection: (arg: PostToConnectionRequest) => { promise(): Promise<unknown> };
+    postToConnection(args: PostToConnectionCommandInput): void;
```

Comment:
Shouldn't this rather be?
```suggestion
    postToConnection(args: PostToConnectionCommandInput): Promise<PostToConnectionCommandOutput>
```
If so this covers up an issue in socket.ts.

```ts
export interface ConfigWithHandler extends Config {
  handler: Pick<ApiGatewayManagementApi, 'postToConnection'>;
}
```
would be a clean alternative as well.

Reply:
Ah, you are right. I copy-pasted the type from the callback version while I should have used the Promise version. The `Pick` approach looks better, though, as it keeps the polymorphism.

Reply:
done